app/models/participant_profile/ecf.rb

```diff
@@ -67,7 +63,7 @@ def self.unfinished_with_billable_declaration(cohort:, restrict_to_participant_i
       .where("participant_declarations.state IN (?) AND declaration_type != ?", Finance::StatementLineItem::BILLABLE_STATES, "completed")
       .where.not(id: completed_billable_declarations.select(:participant_profile_id))
       .distinct
-
+    query = query.where(type: "ParticipantProfile::ECT") if FeatureFlag.active?(:closing_2022)
```

Comment:
I think last time we agreed we are going to add guard clauses to where this is relevant, unfortunately this is used in the API for change schedule, and we already implemented it by adding a guard clause not a feature flag here: https://github.com/DFE-Digital/early-careers-framework/blob/e4e99184e8a320a3b27e5efc70ab1e0405e71ece/app/services/change_schedule.rb#L81 I don't we can use feature flags as this will reverse our already live implementation?